File: Client/Angular/pali-canon/src/app/quote.ts
export class Quote {
    id: number;
    text: string;
    title: string;
    author: string;
    chapter: number;
    verse: number;
    verseLast: number;
    book: string;
    bookCode: string;

    get verseDisplay (): string{
    
      if(this.verseLast === null)
        return this.verse + "";
      else
        return this.verse + "-" + this.verseLast;
      
    }

  }

File: Api/PaliCanon.Model/Quote.cs
namespace PaliCanon.Model
{
    public class Quote
    {
        public string Nikaya { get; set; }
        public string Book { get; set; }
        public string BookCode { get; set; }
        public string ChapterTitle { get; set; }
        public int ChapterNumber { get; set; }
        public string Author { get; set; }
        public string Citation { get; set; }
        public int VerseNumber { get; set; }
        public int? VerseNumberLast { get; set; }
        public string Text { get; set; }
    }
}
